db/migrate/20160412102905_create_container_deployment_custom_settings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContainerDeploymentCustomSettings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_custom_settings do |t|
+      t.string :path
+      t.text :value
+      t.belongs_to :container_deployment, :index => {:name => 'deployment_settings'}, :type => :bigint
```

Comment:
The name should be at least somewhat consistent with other index names, so starts with `index_` and has something about the table name in it at a minimum...something like `index_container_deployment_settings_on_name`?